index.html

```diff
@@ -74,7 +74,7 @@ <h1>Entries</h1>
           </div>
         </div>
         <div class="column-full">
-          <ul class="domCreation">
+          <ul class="domEntry">
```

Comment:
Why not use the name I gave you? `entry-list`. This still doesn't describe its purpose.